codegen/src/main/resources/core/src/ContractsApi.mustache

```diff
@@ -18,6 +18,7 @@ import javax.ws.rs.core.MediaType
 @OpenAPIDefinition(
 info = Info(
 title = "Web3j OpenApi",
+version = "{{version}}",
```

Comment:
Is that the web3j OpenAPI version or a user-defined version?

There is no point in putting here the web3j OpenAPI version IMO.

Reply:
It is for the swagger specs file to be valid when generating a client in Javascript for example.

Reply:
It is possible to leave it without a version but the user will need to add the `skip-validation` flag or something like that or else he will get an error. So, I think it is better to add it there.

Reply:
Sorry but you didn't answer my question, you answered something different.

Reply:
Ah sorry, it is the Web3j OpenAPI version